bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccBindingConstants.java

```diff
@@ -64,7 +66,7 @@ public class EvccBindingConstants {
     public static final String CHANNEL_LOADPOINT_VEHICLE_PRESENT = "vehiclePresent";
     public static final String CHANNEL_LOADPOINT_VEHICLE_RANGE = "vehicleRange";
     public static final String CHANNEL_LOADPOINT_VEHICLE_SOC = "vehicleSoC";
-    public static final String CHANNEL_LOADPOINT_VEHICLE_TITLE = "vehicleTitle";
+    public static final String CHANNEL_LOADPOINT_VEHICLE_NAME = "vehicleName";
```

Comment:
No need to rename the channel because of an evcc API change. This is a breaking change for users, and I would prefer to avoid that.

Changing the binding code to reflect evcc API changes is fine IMO.

Reply:
It is a breaking change - see above